opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -1121,6 +1167,88 @@ namespace detail {
         }
     }
 
+    template <class Grid, class Implementation>
+    void BlackoilModelBase<Grid, Implementation>::solveWellEq(const std::vector<ADB>& mob_perfcells,
+                                                              const std::vector<ADB>& b_perfcells,
+                                                              SolutionState& state,
+                                                              WellState& well_state)
+    {
+        V aliveWells;
+        const int np = wells().number_of_phases;
+        std::vector<ADB> cq_s(np, ADB::null());
+
+        int          it  = 0;
```

Comment:
Weird spacing here? Also, the variable should be defined closer to where it is needed (the `while` loop).